include/wx/generic/grid.h

```diff
@@ -3467,5 +3484,63 @@ extern const int wxEVT_GRID_CHANGE_SEL_LABEL;
 
 #endif
 
+#if wxUSE_ACCESSIBILITY
```

Comment:
Just curious: does this class have to be public? It doesn't seem to be documented, so perhaps it's not supposed to be used by the code outside the library? And in this case, could it please be moved to `include/wx/generic/private/grid.h` or even some new `include/wx/generic/private/gridaccessible.h` to reduce physical coupling?

Reply:
Done; it's in `.../generic/private/grid.h` now